src/BenchmarksServer/hello_world.sql

```diff
@@ -0,0 +1,32 @@
+PRAGMA journal_mode = 'wal';
+
+CREATE TABLE world (
+    id INTEGER NOT NULL PRIMARY KEY,
+    randomNumber INTEGER NOT NULL DEFAULT 0
+);
+
+.system echo "BEGIN;" > temp.sql
+.system pwsh /C "1..10000 | %{ 'INSERT INTO world (randomNumber) VALUES (abs(random() % 10000));' >> temp.sql }"
+.system echo "COMMIT;" >> temp.sql
+.read temp.sql
+.system pwsh /C "rm temp.sql"
```

Comment:
PowerhShell is an acceptable cross-platform procedural language for SQLite, right? 😉

Reply:
I have to check where to move the sqlite binary file, the BenchamrkServer folder is also legacy.

Reply:
We could also create it lazily in C# if that would be easier than keeping a binary file around.

Reply:
That might be better. So when sqlite is chosen, on the startup run the code?

Reply:
Can't be a migration since it should work with ADO.NET/Dapper/EF which are supported here